src/ElmTestRunner/Reporter/Exercism.elm

```diff
@@ -109,14 +110,40 @@ encodeExercismResult { name, taskId, status, message, output } =
         ]
 
 
+coverageReportToString : CoverageReport -> Maybe String
+coverageReportToString coverageReport =
+    case coverageReport of
+        Test.Coverage.NoCoverage ->
+            Nothing
+
+        Test.Coverage.CoverageToReport r ->
+            Just (Test.Coverage.coverageReportTable r)
+
+        Test.Coverage.CoverageCheckSucceeded _ ->
+            {- Not reporting the table to the Exercism stdout (similarly to the
+               Console reporter) although the data is technically there.
+               We keep the full data dump for the JSON reporter.
+            -}
+            Nothing
+
+        Test.Coverage.CoverageCheckFailed r ->
+            Just (Test.Coverage.coverageReportTable r)
+
+
 toExercismResult : TestResult -> ExercismResult
 toExercismResult testResult =
     case testResult of
-        TestResult.Passed { labels } ->
+        TestResult.Passed { labels, successes } ->
             { name = extractTestName labels
             , taskId = extractTaskId labels
             , status = "pass"
-            , message = Nothing
```

Comment:
exercism message field is to be provided only in the case of an error

Reply:
Where could I put the coverage table then? Should that just be ignored for the Exercism reporter?

Reply:
Exercism changes reverted in 4259c50